src/bundler.ts

```diff
@@ -110,11 +112,23 @@ export class Bundler {
    * Analyze a URL using the given contents in place of what would otherwise
    * have been loaded.
    */
-  async analyzeContents(url: ResolvedUrl, contents: string): Promise<Document> {
+  async analyzeContents(
+      url: ResolvedUrl,
+      contents: string,
+      permanent?: boolean): Promise<Document> {
```

Comment:
add documentation of `permanent`

Reply:
Added detailed comment in commit 7f1e8a27020c959c850e1a0af2b376207987da46